.travis.yml

```diff
@@ -4,11 +4,15 @@ env:
 
 language: php
 php :
+  - 5.5
   - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
 
 before_script:
   - composer self-update
-  - composer install --prefer-source --no-interaction --dev
+  - composer update --prefer-source --no-interaction --dev
```

Comment:
updating dependencies every time is not a good practice, as there could be unexpected changes in related libs and everything will break once you update them during tests. 
better to make it a scheduled process, like once per 1-2 month and then test and commit composer.lock

Reply:
so please, also revert composer.lock into previous state

Reply:
You're right, but you have consider the following issue:
In ```php-5.5``` test in Travis CI build, it will not be built successfully because of the locked dependencies. And it cannot test that correctly.
That's why I use the ```composer update``` command in the ```.travis.yml```.

Reply:
ok, makes sense. how about leaving composer.lock as is (with updated libs), but return composer install in `travis.yml`

Reply:
@apollonin, thank you for your reply.
It will be good approach :+1:.
BTW, the ```--dev``` option in ```composer``` command is deprecated.
Please see the previous [Travis CI build log](https://travis-ci.org/apollonin/numphp/builds/354990523).

Thanks.